### PR TITLE
[coolprop] Fix usage, re-fix installation

### LINKED ISSUE #18281: Fail to link CoolProp
**Describe the bug**
I have successfully downloaded and built CoolProp using vcpkg install coolprop. But when I tried to use this pkg, MSVC reported a LINK error saying that function PropsSI not found. It was built as dynamic lib.
Besides, I want to know how to static build this pkg using vcpkg?

**Environment**
- OS: WIN10
- Compiler: MSVS 2019

**To Reproduce**
1. ./vcpkg install coolprop
2. Add following code in main.cpp
#include "CoolPropLib.h"
#include <iostream>

int main()
{
    std::cout << PropsSI("T", "P", 101325, "Q", 0, "Water") << std::endl;
    return 1;
}

**Expected behavior**
Should return 1.

**Failure logs** 
Build started...
1>------ Build started: Project: TestProj, Configuration: Debug x64 ------
1>Main.obj : error LNK2019: unresolved external symbol "double __cdecl PropsSI(char const *,char const *,double,char const *,double,char const *)" (?PropsSI@@YANPEBD0N0N0@Z) referenced in function main
1>D:\Desktop\TestProj\x64\Debug\TestProj.exe : fatal error LNK1120: 1 unresolved externals
1>Done building project "TestProj.vcxproj" -- FAILED.

**Additional context**

### PR DESCRIPTION
**Pull request by JackBoosY**
Fix export macro.

Fixes #18281.

### CODE REVIEW

**Comment by JackBoosY** [comment]
@LiuZhexuan Can you please test my changes?

Thanks.

File: ports/coolprop/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "coolprop",
+  "version": "6.4.1",
```
Comment (NancyLi1013) [inline ports/coolprop/vcpkg.json:3]:
```suggestion
  "version-semver": "6.4.1",
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: versions/c-/coolprop.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "e54cb46a29c92624b2ed32088b1a8c2c7e0738b4",
+      "version": "6.4.1",
```
Comment (JackBoosY) [inline versions/c-/coolprop.json:5]:
```suggestion
      "version-semver": "6.4.1",
```

File: versions/c-/coolprop.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "e54cb46a29c92624b2ed32088b1a8c2c7e0738b4",
```
Comment (JackBoosY) [inline versions/c-/coolprop.json:4]:
```suggestion
      "git-tree": "61676b5a1e9b82c4b88fd3bf37b684f592f9103e",
```

**Review by NancyLi1013 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Cool, thanks @JackBoosY :)

**Pull request merged**